app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -61,9 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             @Override
             public void onRefresh() {
                 DataDownload download = new DataDownload();
-                dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
```

Comment:
In LocationsFragment.java,when we are calling "download.downloadMicrolocations();"
it will call 
"public void downloadMicrolocations() {
        client.getOpenEventAPI().getMicrolocations(Urls.EVENT_ID, new MicrolocationListResponseProcessor());
}"
and we have defined to clear old values and insert new values @onSuccess 
" dbSingleton.clearDatabase(DbContract.Sessions.TABLE_NAME);
dbSingleton.insertQueries(queries);"